src/ResourceManager/Websites/Commands.Websites/Commands.Websites.csproj

```diff
@@ -134,7 +133,6 @@
     <Compile Include="Cmdlets\StartAzureWebApp.cs" />
     <Compile Include="Cmdlets\StopAzureWebApp.cs" />
     <Compile Include="Cmdlets\AppServicePlan\GetAzureAppServicePlan.cs" />
-    <Compile Include="Cmdlets\AppServicePlan\GetAzureAppServicePlanMetrics.cs" />
```

Comment:
we will re enable them as we are working on a fix in backend. we just wanted to ensure these cmdlets are not enabled. Hence not planning to remove the files.